config/manifests/kustomization.yaml

```diff
@@ -1,8 +1,11 @@
 # These resources constitute the fully configured set of manifests
 # used to generate the 'manifests/' directory in a bundle.
-
```

Comment:
Why are we removing the apiVersion & Kind, Aren't they necessary?

Reply:
no, they are not. This is how the scaffolding was created earlier. As it is revert git is reverting everything. They get populated if and only if we change the file. Which was the case when we added the fusion.

Reply:
ack